### PR TITLE
feat: implement the dumb array parser

### PR DESCRIPTION
**Pull request by kkweon**
- it will use the first element to infer the type

### CODE REVIEW

File: src/number_utils.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// If there is no remainder when x % 1, it's an integer.
```
Comment (TangoEnSkai) [inline src/number_utils.ts:1]: wow, never thought about doing it in this way, but it makes sense 👍

File: src/parse_proto_field.ts [hunk 2]
```diff
@@ -52,21 +49,31 @@ export function parseProtoField(
...
+        protoField.setType(createEmptyProtoType(fieldName))
       } else if (Array.isArray(obj)) {
-        // TODO(kkweon): Handle array
+        if (obj.length === 0) {
+          protoField.setType(createEmptyProtoType(fieldName))
+          protoField.setRepeated(true)
+        } else {
+          // use the first element to infer the type
+          // TODO(kkweon): Merge messages when multiple different type appears in the array.
+          const elementProtoField = parseProtoField(obj[0], fieldName, fieldTag)
```
Comment (TangoEnSkai) [inline src/parse_proto_field.ts:60]: nit: just curious if using magic number in code blocks is okay/common. 
e.g. `obj[0]` v.s. `obj[start]`

Comment (kkweon) [reply src/parse_proto_field.ts:60]: That's why it's "dumb" version. It will be revisited once there is a logic to merge two different types.

Comment (TangoEnSkai) [reply src/parse_proto_field.ts:60]: thanks for the comment. understood 👍

**Pull request merged**